R-packages/evalcast/R/download_signals.R

```diff
@@ -64,7 +64,17 @@ download_signals <- function(...,
   for (i in seq_along(msg)) message(msg[i])
   
   out <- base::suppressMessages({covidcast_signals(...)})
+  empty_signals <- purrr::map_lgl(out, ~nrow(.x) == 0 || is.null(.x))
```

Comment:
Should the conditions here be reversed for lazy evaluation purposes?  I assume `nrow(NULL)` is well defined, but it seems better to skip that.